book/requirements.txt

```diff
@@ -0,0 +1,13 @@
+geopandas==0.10.2
+matplotlib==3.5.1
+numpy==1.21.2
+pandas==1.3.5
+plotly==5.3.1
+pystac-client==1.1.0
+requests==2.26.0
+rasterio==1.2.10
+scikit-image==0.18.3
+scikit-learn==0.24.2
+stackstac==0.4.0
+xarray==0.19.0
```

Comment:
`xarray` graduated to releases based on the current year over 2 years ago, and `0.19.0` is nearly 3 years old, so I suggest we bump this to the latest. In general, I suggest we bump as many of these dependencies to the latest possible (whatever can be fully resolved in conjunction).

Reply:
this file is not part of the github action